walletcontroller/client.go

```diff
@@ -66,7 +68,7 @@ func NewRpcWalletControllerFromArgs(
 ) (*RpcWalletController, error) {
 
 	connCfg := &rpcclient.ConnConfig{
-		Host:                 host,
+		Host:                 host + "/wallet/" + walletName,
```

Comment:
I would say it would be good, if ` walletName` is empty string for this to be just `host`

Reply:
https://github.com/babylonlabs-io/btc-staker/pull/25/commits/c6d6b37731eb0e3f54cb451399f6c6d2202f27ef